.github/scripts/validate_binaries.sh

```diff
@@ -0,0 +1,8 @@
+
+if [[ ${MATRIX_PACKAGE_TYPE} = "conda" ]]; then
+    conda install -y torchdata -c ${PYTORCH_CONDA_CHANNEL}
+else
+    pip install ${PYTORCH_PIP_PREFIX} torchdata --extra-index-url ${PYTORCH_PIP_DOWNLOAD_URL}
+fi
+
+python  ./test/smoke_test/smoke_test.py
```

Comment:
Can we do `python ./test/smoke_test/smoke_test.py --no-s3` when ${MATRIX_PACKAGE_TYPE} is `conda` on Windows?

Reply:
Resolved as I just pushed the change to your branch.